.travis_update_ghpages.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Only publish from the main repository's master branch
+REPO_NAME="HydraCG/Specifications"
+if [ "$TRAVIS_REPO_SLUG" != "$REPO_NAME" ] || [ "$TRAVIS_BRANCH" != "master" ] || [ "$TRAVIS_PULL_REQUEST" != "false" ]; then exit; fi
+echo -e "Generating gh-pages...\n"
+
+# Checkout the gh-pages branch
+REPO_PATH=$PWD
+pushd $HOME
+git clone --quiet --branch=gh-pages https://${GH_TOKEN}@github.com/$REPO_NAME gh-pages 2>&1 > /dev/null
```

Comment:
Why not use the [`docs` folder approach](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch) instead of the (imho) hacky `gh-pages` branch approach?